fahari/mle/models.py

```diff
@@ -0,0 +1,85 @@
+from django.contrib.gis.db import models
+from django.urls.base import reverse
+from django.utils import timezone
+
+from fahari.common.models import AbstractBase, Facility
+
+
+class FacilityNetworkCondition(AbstractBase):
+    """Facility network conditions."""
+
+    facility = models.ForeignKey(Facility, on_delete=models.PROTECT)
+    TRUE = "TRUE"
+    FALSE = "FALSE"
+    STATE_INDICATORS = [
+        (TRUE, "True"),
+        (FALSE, "False"),
+    ]
+    has_network = models.CharField(max_length=50, choices=STATE_INDICATORS, default=TRUE)
+    has_internet = models.CharField(max_length=50, choices=STATE_INDICATORS, default=TRUE)
```

Comment:
This too. Change this to a boolean field.

Reply:
fixed